rtc/configuration.md

```diff
@@ -8,361 +8,359 @@ keywords: rtc configuration
-```
-
-### Example of providing encryption details from your server
-
-```js
-tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugins: 'rtc',
-  rtc_encryption_provider: ({documentId, sessionId, keyHint}) => {
-    return fetch('http://yourserver/getKey', {
-      method: 'POST',
-      body: JSON.stringify({ documentId, sessionId, keyId: keyHint })
-    })
-    .then(({keyId, secret}) => ({ key: secret, keyHint: keyId });
-  }
-})
-```
-
-## `rtc_token_provider`
-
-The RTC plugin and service uses [JWT]({{site.baseurl}}/rtc/jwt-authentication/) to authenticate the user. This token should include a unique user ID and a relative expiration time. This provider function will be called multiple times to refresh the token if it's about to expire. For production usage, the token provider should be a dynamic request that produces a new JWT token with an updated `exp` claim.
+The RTC plugin and service uses [JWT]({{site.baseurl}}/rtc/jwt-authentication/) to authenticate the user. The user's token should include a unique user ID and a relative expiration time. This provider function will be called multiple times to refresh the token if it's about to expire. For production usage, we recommend the token provider be a dynamic request that produces a new JWT token with an updated `exp` claim.
 
 **Type:** `Function`
 
 **Required:** yes
 
-### Required JWT claims
+#### Required JWT claims
 
 | Field | Type | Description |
 |-------|:----:|-------------|
-| `sub` | `string` | The unique user ID (If `sub` is the same for two clients, the server should trust them as if they are the same user). |
+| `sub` | `string` | The unique user ID (If `sub` is the same for two clients, the server will trust them as if they are the same user). |
 | `exp` | `integer` | The timestamp when the token expires. |
 
-### Return fields for `rtc_token_provider`
+#### Return fields for `rtc_token_provider`
 
 | Field | Type | Description |
 |-------|:----:|-------------|
-| `token` | `string` | A generated JWT token. This token should be signed with the private key. |
+| `token` | `string` | A generated JWT token. This token should be signed with a private key as described in [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/). |
+
+### Minimum configuration example
 
-### Example of providing a static JWT token
+#### Static configuration
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
+  selector: 'textarea',
   plugins: 'rtc',
+  rtc_document_id: "your-document-id",
+  rtc_encryption_provider: () => Promise.resolve({ key: "your shared encryption key", keyHint: "not used" }),
   rtc_token_provider: () => Promise.resolve({ token: "your-jwt-token" })
 })
 ```
 
-### Example of providing a JWT token from your server
+#### Dynamic configuration that fetches encryption keys and tokens from a server
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
```

Comment:
Why was this comment removed? As this is supposed to be included in every example since it'll need changing based on the integration.

Reply:
I thought `textarea` was default? I just saw in the API docs it is listed as required, so I guess I need to put them all back 🤔

Reply:
oh you just meant the comment, sorry

I dunno, it just seemed superfluous.

Reply:
We also don't include it in a _lot_ of the other examples (113 of 541 references in search results)

Reply:
maybe leave it as removed for this PR. When we have a better solution or decide on the standard approach, I'll do a find and replace across the repo.